app/helpers/check_functions.py

```diff
@@ -88,3 +89,20 @@ def get_and_check_icon(icon_set, icon_name):
         logger.error("The icon doesn't exist: %s", path)
         abort(404, "Icon not found in icon set")
     return icon
+
+
+def check_if_descripton_file_exists(icon_set):
```

Comment:
It might be OK for certain icon sets to not have a description file.
Maybe we can find an easy and generic way to track, which icon sets should have a description file and which shouldn't.
And only raise the error, when there's no description file even if one was expected to exist

Reply:
If an icon set does not have a description file the request 
`<host>/sets/<icon_set_name>` will not return a description url so I don't think that this is necessary
`{
  "colorable": true, 
  "description_url": null, 
  "icons_url": "http://localhost:5000/sets/default/icons", 
  "name": "default", 
  "success": true, 
  "template_url": "http://localhost:5000/sets/{icon_set_name}"
}
`